app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -238,6 +241,9 @@ public void onShowTabsButtonClicked() {
 
     @OnClick(R.id.page_toolbar_button_show_overflow_menu)
     public void onShowOverflowMenuButtonClicked() {
+        if (watchlistTooltip != null) {
```

Comment:
do we also want to check if it's `isShowing()`?

Reply:
Yes and done.